electron_app/helpers/enviroment.js

```diff
@@ -15,6 +15,7 @@ const constants ={
   IPC_DOWNLOAD_CANCEL: 'download-cancel',
   IPC_GOOGLE_LOGOUT: 'google-revoke-auth',
   GOOGLE_LOGOUT_URL: 'https://accounts.google.com/logout?continue=http://google.com',
+  USER_AGENT: 'FireCloudExplorer/0.1.0 Alpha Release/Electron',
```

Comment:
I guess there's no way to get this version number from `package.json`? We should probably start keeping a list of all the places that need to be updated when the version number changes, since it's a manual process, and it's duplicated in a few different places.

Reply:
Yes I did a research to find how to use the version number in package.json and I didn't find something useful

Reply:
Sorry, there is a way. I'm going to implement that.

Reply:
Sorry, there is a way. I'm going to implement that. When I investigated for using the content of package.json was in the case to use it inside the Angular app.